examples/gatsby-using-plugin/.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I think we should just update the default example to use the plugin, rather than have two